supervised/utils/additional_metrics.py

```diff
@@ -137,6 +137,79 @@ def binary_classification(target, predictions, sample_weight=None):
 
         threshold = float(max_metrics["accuracy"]["threshold"])
 
+        accu_threshold_details = {
```

Comment:
please change the variable name to `metrics_at_accuracy_threshold`

Reply:
Okay